.dockerignore

```diff
@@ -0,0 +1,132 @@
+### Flask ###
```

Comment:
Using your default `.gitignore` as a starting point for this makes sense, but it's worth noting the syntax isn't always identical (e.g. [see this article](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/)) so this might not always do exactly as you expect.

In your case I think all of the relevant ignored files to us are OK, but I'd also suggest in future trying to keep this more specific to your own system to make it easier to reason about what's included & why.

Reply:
I have make a note of it and in future I will only add specific files to ignored files.